packages/compass-e2e-tests/helpers/commands/set-connect-form-state.ts

```diff
@@ -1,8 +1,29 @@
+import _ from 'lodash';
 import { expect } from 'chai';
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 import type { ConnectFormState } from '../connect-form-state';
 
+async function waitForElementAnimations(browser: CompassBrowser, element: any) {
+  let previousResult = {
+    ...(await element.getLocation()),
+    ...(await element.getSize()),
+  };
+  await browser.waitUntil(async function () {
```

Comment:
actually I just realised that waitUntil also exists on element, but then we use beowser.pause which does not anyway, so whatever.